Adding the core strategy (#12)
esdc-devcop/strategy: README.md

# ESDC's Developer's Community of Practice

## Help out

Feel free to fork and submit a PR to any of our repos, or ask to join as a Member.  
Join our conversation in Slack, we have a channel in the [GC DevOps League](https://gcdevopsleague.slack.com) [#esdc-devcop](https://gcdevopsleague.slack.com/messages/CKS45C1ST/).  

Also, [vote on Issues](https://github.com/esdc-devcop/strategy/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) (with a `+1` reaction) that you think we should put a focus on doing first!

## Strategy

Each month we will focus on one core [topic](https://github.com/esdc-devcop/strategy/issues?q=is%3Aopen+is%3Aissue+label%3Atopic) that could be actioned and implemented by a team with less than half a day's work. 
Each month will build on the previous where it will lead teams to developing applications better and faster though increasing automation and reducing feedback loops.

> Throughout all of this wee will need to keep in mind that their are many types of technology being used in applications that will need these implementation;

Reviewer: ```suggestion
Throughout all of this, we will need to keep in mind that their are many types of technology being used in applications that will need these implementations;
```

> so our solutions need to be non-tool dependant but offer suggestions and recommendations for tools.

Reviewer: ```suggestion
So our solutions need to be non-tool dependant but offer suggestions and recommendations for tools.
```

We also need to keep in consideration that there are ~140 development teams in IITB and we will not have the capacity to help them all one-by-one.
So we will need to lay out plans that can be adopted without our intervention.

There are four key parts to making this all work:

- [Info Sessions](#info-sessions)
- [Hands on Sessions](#hands-on-sessions)
- [Management Support](#management-support)
- [Community Building & Engagement](#community-building-engagement)

### Info Sessions

Holding monthly sessions that will introduce the monthly topic and layout the plan of how to implement the topic.

> These sessions will be hosted similar to 'Ted Talks' or 'Coffee and Learn' with a short Q&A period.

Reviewer: ```suggestion
These sessions will be hosted similar to 'TED Talks' or 'Coffee and Learn' with a short Q&A period.
```

These sessions should also regularly be bringing in members from other areas of IITB to give a less than 5 min update on what they are working on. 
(Areas to conceder: Testing, Accessibility, Security, QA, DBA)

### Hands on Sessions

Throughout the month, we will be planning 'Workshops' or 'Hackathons' that are targeting teams to help them implement that topic of the month.
We will look at using experts and the community as a whole to help with these events.

### Management Support

We will be providing support to teams that are keen on implementing these topics but are receiving resistance from management.
Working with management on what they perceive to be preventing them from accepting the implementation to demonstrate how they can do it now.

### Community Building & Engagement

> We will keep a focus on keeping the community engaged and involved in the various activities though social media and advertising methods.

Reviewer: ```suggestion
We will also focus on keeping the community engaged and involved in the various activities through social media and advertising methods.
```
